MeetHut.Backend/ClientApp/src/app/auth/auth.module.ts

```diff
@@ -13,6 +13,44 @@ import {
   SocialLoginModule
 } from 'angularx-social-login';
 import { SharedModule } from '../shared/shared.module';
+import { ParameterService } from '../shared/services';
+
+const socialConfigFactory = (parameterService: ParameterService) =>
+  new Promise((resolve) => {
+    parameterService.getAll().then((res) => {
+      const providers = [];
+
+      if (res) {
+        // Google
+        const googleClientIdParam = res.find(
+          (x) => x.key === 'Google.ClientId'
+        );
+        if (googleClientIdParam && googleClientIdParam.value) {
+          providers.push({
+            id: GoogleLoginProvider.PROVIDER_ID,
+            provider: new GoogleLoginProvider(googleClientIdParam.value)
+          });
```

Comment:
Looks like the redirectUri is missing here, isn't it?

Reply:
Here not necessary. The Google Application contains this Uri on the server side.

Reply:
Ok, got it, thanks.